### DISCUSSION TITLE
[Bug] AC M3 Max, only global wait times possible?

### DISCUSSION THREAD

**Discussion by gaby64** [sn4k3/UVtools#938] [Q&A]
### System

```Text
UVtools v4.4.3 X64
Operative system: Microsoft Windows 10.0.19045 X64
Graphic card: 
Processor: Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
Processor cores: 8
Memory RAM: 13.37 / 15.43 GB
Runtime: win10-x64
Framework: .NET 6.0.33
AvaloniaUI: 11.1.3
OpenCV: 4.9.0

Sreens, resolution, working area, usable area:
1: 3840 x 2160 @ 225% (Primary) (On this)
    WA: 3840 x 2070    UA: 1707 x 920

Path:       C:\Program Files\UVtools\
Executable: C:\Program Files\UVtools\UVtools.exe
Loaded file: SquareKnob_M3.pm3m [Version: 516] [Class: AnycubicFile]
```

### Printer and Slicer

- Printer: Anycubic Photon M3 Max
- Slicer: Lychee Slicer 7

### Description of the bug

Try to set different wait before cure times for bottom layers, only applies globally to all layers.

### How to reproduce

Tools->Edit print parameters->set wait time for layer with propagate unchecked.
Suggestions->Wait time before cure

**Comment by sn4k3** [comment]
Not possible with your printer or any anycubic. It only have a global setting.
